src/index.html

```diff
@@ -7,8 +7,29 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="0">
+    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html"><img src="images/logo.png" alt="logo"></a>
+      <nav>
+        <ul>
+          <li><a class="nav__link is-active" href="#Apple">Apple</a></li>
+          <li><a class="nav__link" href="#Samsung">Samsung</a></li>
+          <li><a class="nav__link" href="#Smartphones">Smartphones</a></li>
+          <li>
+            <a class="nav__link" href="#Laptop"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
```suggestion
            <a
              class="nav__link"
              href="#Laptop"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Don't forget about correct indents, also if you have 3 or more attributes than move each attribute to a new line

Reply:
@nazarmatsevych corrected

Reply:
Not fixed